cmd/rook/ceph/cleanup.go

```diff
@@ -40,22 +46,40 @@ var (
 
 var cleanUpCmd = &cobra.Command{
 	Use:   "clean",
+	Short: "Starts the cleanup process",
+}
+
+var cleanUpDisksCmd = &cobra.Command{
+	Use:   "disks",
```

Comment:
Aren't we cleaning subvolumes not disks?

Reply:
We already had a `ceph clean` clean to clean up the data on the disks. 

I've extend to `clean` command with more sub commands. 

`ceph clean disks` will be use the data cleanup job. This is an existing behavior. 

`ceph clean CephFSSubVolumeGroup` will clean up the subvolumegrous.

Reply:
`ceph clean disks` is called by which container?